crates/terminal/src/lib.rs

```diff
@@ -91,6 +91,15 @@ macro_rules! error {
     }};
 }
 
+#[macro_export]
+macro_rules! info {
+    ($highlight:expr, $($arg:tt)*) => {{
+        $crate::ceprint!($crate::colors::bold_cyan(), $highlight);
```

Comment:
Not sure how I feel about `step!` going to stdout while `error!` and `info!` go to stderr. For text processing tools the distinction is clear to me: stdout is for "data" that you might be piping to another command, while stderr is for stuff that is only for human consumption. It isn't obvious how/if that distinction applies here.

Reply:
I sent the update badger to stderr because I explicitly didn't want it mixed up with plugin output that might be getting consumed by a program (e.g. a plugin that outputs JSON).  I can be more explicit and call this `einfo!` or something if you prefer - that might also make it clear the follow-up lines should use `eprintln` not `println`.  Let me know if you want me to make that or a similar change.

Reply:
Ah yes that makes perfect sense for :badger:. `einfo` would make sense to me, especially to help differentiate from `step`.